### DISCUSSION TITLE
CRC OKD: crc-admin-helper-linux version mismatch: v0.0.6 expected but 0.0.6 found in the cache

### DISCUSSION THREAD

**Discussion by syedriko** [okd-project/okd#969] [Q&A] [answered]
I'm getting the error below running the OKD flavor of CRC I grabbed from `https://dl.fedoraproject.org/pub/alt/okd-crc/release/linux-amd64/`
```
time="2021-11-08T11:50:50-05:00" level=debug msg="Running '/home/syedriko/.crc/bin/crc-admin-helper-linux --version'"
time="2021-11-08T11:50:50-05:00" level=debug msg="crc-admin-helper-linux version mismatch: v0.0.6 expected but 0.0.6 found in the cache"
time="2021-11-08T11:50:50-05:00" level=debug msg="unexpected version of the crc-admin-helper executable: crc-admin-helper-linux version mismatch: v0.0.6 expected but 0.0.6 found in the cache"
time="2021-11-08T11:50:50-05:00" level=warning msg="Preflight checks failed during `crc start`, please try to run `crc setup` first in case you haven't done so yet"
```
Is anyone else seeing this? Seems like there's an extra "v" tacked on in front of the expected version. Issue material?

**Comment by gbraad** [comment]
A few weeks ago for CRC, the project of which OKD's is derived, added the `v` to the releases, as can be seen here: https://github.com/code-ready/admin-helper/releases For this we bumped the code to reference this: https://github.com/code-ready/crc/commit/dc60c7bb7eb8ce5a1bc90e0119d08870028426a9

So not sure what is happening with OKD, but for CRC the `v` is now enforced as a tag. Perhaps @cgruver knows what is happening for OKD.

**Comment by cgruver** [comment]
@syedriko When did you download that CRC release?  The current one for OKD should be using v0.0.7 of the crc-admin-helper-linux.

The v0.0.6 release does not work currently because the executable still reports its version as 0.0.6 while the crc binary is looking for v0.0.6.

**Comment by cgruver** [comment, marked as answer]
@syedriko   The latest crc binary with the updated crc-admin-helper-linux failed to push to the fedora server.  It is uploading now.

**Reply by syedriko** [reply]
The latest crc binary does not have this issue. Thank you, @cgruver!
